src/atoms/Layout/index.js

```diff
@@ -124,7 +124,7 @@ Layout.propTypes = {
    *
    * `'bordered-buckets'`: Creates borders around `Col` from medium viewports up.
    */
-  variant: PropTypes.oneOf([ 'bordered-buckets' ]),
+  variant: PropTypes.oneOf([ 'bordered-buckets', 'padding-small' ]),
```

Comment:
This seems hyper specific but given the problem you need to solve at hand, this seems like the easiest way to solve it.